SprintCleaning/Assets/Scripts/Garbage.cs

```diff
@@ -25,12 +27,18 @@ private void OnTriggerEnter(Collider other)
         {
             GameObject player = other.transform.parent.gameObject;
             _garbageAudio = player.GetComponent<AudioSource>();
-
+            _garbageAudio.PlayOneShot(impact, 1F);
+            if (_obstacle)
+            {
+                player.GetComponent<DirtinessManager>().AddDirtiness(30);
```

Comment:
Dirtiness is an old game mechanic, so it'd be better to make a new script and copy whatever code is necessary, then delete everything related to dirtiness. Reusing a mechanic we're going to move could cause confusion